app/lib/bk/compat/parsers/bitbucket/services.rb

```diff
@@ -20,7 +17,7 @@ def load_services!(global_services)
 
           plugin_config = { 'run' => 'app' }
 
-          @services = BK::Compat::CircleCISteps::CircleCIStep.new(
+          @services = BK::Compat::CircleCIStep.new(
```

Comment:
why is BitBucket using the `CircleCIStep`, I strongly believe this was an overzealous copy-paste on my part:

```suggestion
          @services = BK::Compat::CommandStep.new(
```

Reply:
Great pickup - I've rolled this through as a `CommandStep`. Effectively was only really using the base class parameters anyway 😉